src/app/theme/SplashScreenService.ts

```diff
@@ -29,12 +41,41 @@ export const showSplashScreen = (
     }
   )
 
+  if (autoHideTimer) {
+    clearTimeout(autoHideTimer)
+  }
+
+  autoHideTimer = setTimeout(() => {
```

Comment:
I am not sure about it, but I think we will autohide the `secure_background`. This one should not use autohide.

Reply:
- [ ] to update